generator/integration_tests/tests/golden_kitchen_sink_stub_test.cc

```diff
@@ -478,7 +477,7 @@ TEST_F(GoldenKitchenSinkStubTest, AsyncStreamingWriteRead) {
 
   auto mock_cq = std::make_shared<MockCompletionQueueImpl>();
   grpc::CompletionQueue grpc_cq;
-  EXPECT_CALL(*mock_cq, cq).WillRepeatedly(ReturnRef(grpc_cq));
+  EXPECT_CALL(*mock_cq, cq).WillRepeatedly(Return(&grpc_cq));
```

Comment:
I suspect this test (and all the other tests in this file) can be changed to `Return(nullptr)`.